eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+    "root": true,
+    "extends": "airbnb-base",
```

Comment:
@OKiMaureen  You are to use `airbnb` and not `airbnb-base`

Reply:
okay